src/kvstore/RocksEngine.cpp

```diff
@@ -425,5 +430,48 @@ ResultCode RocksEngine::flush() {
     }
 }
 
+ResultCode RocksEngine::createCheckpoint(const std::string& path) {
```

Comment:
We could create multi snapshots under the path.   So we need to pass a snapshot name here.

Reply:
Good ! changed the path struct as below 
     --FLAGS_data_path
      *  --nebula
      *     --space0
      *         --data
      *         --wal
     *   --checkpoint
     *      --checkpoint_1
     *        --nebula
     *          --space1
     *              --data